containers/docker/tools/disable_services.sh

```diff
@@ -0,0 +1,14 @@
+SERVICES=/etc/service/*
+
+for f in $SERVICES
+do
+	service=$(basename "$f")
+	if [ "$service" = "syslog-ng" ]; then continue; fi;
+	if [ "$service" = "syslog-forwarder" ]; then continue; fi;
+	if [ "$service" = "ssh" ]; then continue; fi;
+	if [ "$service" = "cron" ]; then continue; fi;
+	if ([ -d /etc/service/$service ] && [ ! -f /etc/service/$service/down ]); then
+		echo "Creating down file for '$service'"
+		touch /etc/service/$service/down
```

Comment:
This script is runned when creating the docker image. 

It ensure all services are disabled and won't start automatically, because some services are still unconfigured and would spam logs with errors. 

They are enabled when they are started manually with `restart_service` function from setup script. That's why I've added some `restart_service` calls in the project.